src/GraphControl/Control/Grapher.h

```diff
@@ -148,6 +148,104 @@ public
         Platform::String ^ ConvertToLinear(Platform::String ^ mmlString);
         void PlotGraph();
         void AnalyzeEquation(GraphControl::Equation ^ equation);
+       // We can't use the EValTrigUnitMode enum directly in as the property type because it comes from another module which doesn't expose
+        // it as a public enum class.  So the compiler doesn't recognize it as a valid type for the ABI boundary.
+        property int TrigUnitMode
+        {
+            void set(int value)
+            {
+                if (value != (int)m_solver->EvalOptions().GetTrigUnitMode())
+                {
+                    m_solver->EvalOptions().SetTrigUnitMode((Graphing::EvalTrigUnitMode)value);
+                    UpdateGraph();
+                }
+            }
+
+            int get()
+            {
+                return (int)m_solver->EvalOptions().GetTrigUnitMode();
+            }
+        }
+
+        property double XAxisMin
+        {
+            double get()
+            {
+                return m_graph->GetOptions().GetDefaultXRange().first;
+            }
+            void set(double value)
+            {
+                std::pair<double, double> newValue(value, XAxisMax);
```

Comment:
Feels odd that we are doing this directly in the setter instead of how the other properties have a corresponding call back in the cpp file such as OnForceProportionalAxesChanged.